src/udiskslinuxmanager.c

```diff
@@ -746,15 +762,9 @@ handle_mdraid_create (UDisksManager         *_object,
       g_free (escaped_device);
     }
 
-  /* Create the array... */
-  str = g_string_new ("mdadm");
-
   /* we have name from the user */
   if (strlen (arg_name) > 0)
-    {
-      array_name = udisks_daemon_util_escape (arg_name);
-      g_string_append_printf (str, " --create \"%s\"", array_name);
-    }
+      array_name = g_strdup (arg_name);
```

Comment:
Check the `else` part here -- `arg_name` can be empty string and we need to get the _/dev/mdX_ "name" using `udisks_daemon_util_get_free_mdraid_device` (because libblockdev/mdadm needs some "name" when creating the array).